src/pybind/mgr/dashboard_v2/requirements.txt

```diff
@@ -30,3 +30,4 @@ tempora==1.10
 tox==2.9.1
 virtualenv==15.1.0
 wrapt==1.10.11
+requests-aws==0.1.8
```

Comment:
We need to check if this package is available in the several distros (centos, fedora, ubuntu, opensuse, ...) and find its name for each distro, and add to the proper packaging file.

Reply:
as this package contains only one python class to authenticate to S3, I wonder if we can use a different approach than making this a new ceph dependency

Reply:
I dimly recall we had this conversation before. My personal take: if there is already a solution in the form of a Python package, I'm in favor of using it instead of duplicating code/functionality, just to avoid having to deal with packaging dependencies.

Reply:
FYI, This is [`requests-aws`](https://github.com/tax/python-requests-aws/blob/master/awsauth.py#L20-L150). @p-na, can you add @smithfarm  as a reviewer please?

Reply:
Hmm requests-aws is actually BSD licensed. I wonder if it would be appropriate to simply include a copy of that python class in the Ceph git repo? @liewegas , @jcsp - what's your take on adding third-party code?

Reply:
I vote to lift the awsauth.py file into our tree, it's completely within the letter and spirit of the BSD license.  We would just need to cat the LICENSE.txt into the top of the source file.  There's precedent for this with a couple of other BSD-licensed files in the tree.

Reply:
Thanks for your guidance, @jcsp - @p-na can you please take care of that? I suggest to perform the addition of that module as a separate commit, which clearly indicates from where the file was obtained from.

Reply:
Sure, will do. Thanks for your feedback!